src/main/java/com/aws/greengrass/lifecyclemanager/KernelAlternatives.java

```diff
@@ -64,6 +65,7 @@ public class KernelAlternatives {
     private static final String KERNEL_LIB_DIR = "lib";
     private static final String LOADER_PID_FILE = "loader.pid";
     static final String LAUNCH_PARAMS_FILE = "launch.params";
+    public static final String LOGS_DIR = "logs";
```

Comment:
should read the Greengrass log directory, since the customer can configure it, right?

Reply:
Yep, thats right, gonna make that change now